_source/_posts/2021-08-09-aws-toolkit-for-visual-studio.md

```diff
@@ -0,0 +1,415 @@
+---
+layout: blog_post
+title: "Using AWS Toolkit for Visual Studio"
+author: nickolas-fisher
+by: contractor
+communities: [devops, .net]
+description: "Tutorial: Learn how to deploy applications to AWS right from Visual Studio"
+tags: [aws, dotnet, visual-studio]
+tweets:
+- "You don't need to leave Visual Studio to deploy your app to AWS ☁️"
+- "Build, Test, Deploy, right from your IDE‼️"
+- "Learn how to deploy your .Net apps to #AWS from @VisualStudio💥"
+image: blog/featured/okta-dotnet-books-quarter.jpg
+type: conversion
+---
+
+Amazon Web Services is one of the most popular cloud computing platforms on the planet. There's a good chance you will need to work in an AWS environment, which means publishing to one of their web server services like Elastic Beanstalk. The AWS platform is enormous, and this can be an intimidating task for a developer. Luckily, Amazon has released their AWS Toolkit for Visual Studio 2013-2015 and AWS Toolkit for Visual Studio 2017-2019, which make publishing your web applications to Elastic Beanstalk a snap.
+
+In this article, you will build an ASP.Net MVC web application on .Net 5 using Visual Studio 2019. You will secure this application using Okta. Okta makes it easy to create a secure application with their SSO provider. Once your application is complete you will publish it to a production Elastic Beanstalk instance using the AWS Toolkit.
+
+**Table of Contents**{: .hide }
+* Table of Contents
+{:toc}
+
+### What You'll Need
+
+> [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/)
+> [.NET 5](https://dotnet.microsoft.com/download/dotnet/5.0)
+> [An Okta Developer Account](https://developer.okta.com/signup/)
+> [The Okta CLI tool](https://cli.okta.com/)
+> [An AWS Account](https://portal.aws.amazon.com/billing/signup?nc2=h_ct&src=default&redirect_url=https%3A%2F%2Faws.amazon.com%2Fregistration-confirmation#/start)
+
```

Comment:
```suggestion
### What You'll Need

- [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/)
- [.NET 5](https://dotnet.microsoft.com/download/dotnet/5.0)
- [An Okta Developer Account](https://developer.okta.com/signup/)
- [The Okta CLI tool](https://cli.okta.com/)
- [An AWS Account](https://portal.aws.amazon.com/billing/signup?nc2=h_ct&src=default&redirect_url=https%3A%2F%2Faws.amazon.com%2Fregistration-confirmation#/start)

```

Sometimes copying from a Google Doc messes up bullets, oftentimes they just get lost, this time they are `>` characters for some reason, they need to be `*` or `-` chars to render correctly

cc: @havi